project_report/tex/evaluation.tex

```diff
@@ -1 +1,51 @@
 \section{Evaluation}
+The evaluation section describes how the reflections of the goals, set for the
+project, meets the customer demands, as well as how the development process has
+been.
+
+
+\subsection{Evaluation from the customer side}
+The group has managed to meet the requirements, set in the initial requirements
+analysis, at the beginning of the project. The created system is able to control
+the production line, using the dashboard, as well as collect, display and
+store relevant data in a database. The data is used for further analysis, such
+as calculating the OEE and optimal machine speed, as well as estimating the
+error function.
+The design of the dashboard makes it easy to see what is going on the front-page.
+Every item on the front-page is there, nothing is hiding in a sub-menu.
```

Comment:
Consider the following;
 "... is there; nothing is..."
 "... is there, and nothing is..."
 "... is there. Nothing is..."